aggregator/models.py

```diff
@@ -231,6 +236,16 @@ class Meta:
             ),
         ]
 
+    @property
+    def og_image(self):
+        source = requests.get(self.website_url).text
+        soup = BeautifulSoup(source, "html.parser")
+        og_image = soup.find("meta", property="og-image")
+        if not og_image:
+            return None
+
+        return og_image.get("content")
+
```

Comment:
This approach is problematic:
1. Performance It will make request for each 
2. Security:
+ It allows user to make requests to any address, also to internal IPs in cloud - urls should be sanitizd
+